runtime-modules/project-token/src/events.rs

```diff
@@ -32,6 +45,36 @@ decl_event! {
         /// - account
         PatronageCreditClaimed(TokenId, Balance, AccountId),
 
+        /// Revenue Split issued
+        /// Params:
+        /// - token identifier
+        /// - start of the split
+        /// - duration of the split
+        /// - JOY allocated for the split
+        RevenueSplitIssued(TokenId, BlockNumber, BlockNumber, JoyBalance),
+
+        /// Revenue Split issued
+        /// Params:
+        /// - token identifier
+        /// - recovery account for the leftover funds
+        /// - leftover funds
+        RevenueSplitFinalized(TokenId, AccountId, JoyBalance),
+
+        /// Revenue Split issued
+        /// Params:
+        /// - token identifier
+        /// - user account
+        /// - user allocated reserved balance
```

Comment:
> No other term is allowed: liquidity, reserved, reservability, free balance, usability... its all banished!

_Source: https://github.com/Joystream/joystream/issues/3633_

Based on this, please restrain from using terms like `reserved` / `unreserved`. I suggest using `unstaked` instead.